.libres_version

```diff
@@ -0,0 +1 @@
+export LIBRES_VERSION=2.3.10
```

Comment:
And ECL_VERSION?

Reply:
The ECL version is given from the libres repo

Reply:
I think ERT should have its own version. Otherwise, adapting to a new major version of ecl in libres would potentially break master ERT.

Reply:
From what I could see, ERT is using a small number of functions directly from libecl. The most used is util functions, e.g. `from ecl.util.util import BoolVector`. Other than that it's mostly tests that use `ecl` on its own.

I thought that overriding libres' own defined dependencies could cause issues..